CONTRIBUTING.md

```diff
@@ -14,11 +14,22 @@ Stakeholders are able to easily track progression of the project.
 
 ## Our Principles
 
-- We will prioritise reviewing Pull Requests (PRs) over beginning new issues.
-- Where possible, issues will include a user story and product requirements/acceptance criteria as well as links to relevant designs.
-- Where possible, issues will be grouped together according to the product feature they relate to.
-- After two days' effort, an assignee will add a comment to an issue and/or consider raising new issues to clearly indicate progress.
+These are general principles which should be the norm apart from in extreme circumstances.
+
+### Pull Requests
+
 - When feedback is required, an assignee will open a PR with the 'WIP' prefix. See [this ADR](https://github.com/elifesciences/elife-xpub/blob/develop/docs/adr/0001-pull-request-workflow.md).
-- As we facilitate deployment on a per-branch basis, once a PR is merged it is considered 'Done'.
-- If a feature can reasonably be tested using an existing browser test, this will be undertaken before the PR is approved. Any further or additional end-to-end testing will be raised in a new issue with the 'browser test' label and referencing the original issue.
+- The Developer raising the PR will be the one who merges that PR.
+- As deployment is on a per-branch basis, once a PR is merged it is considered 'Done'.
+- If a feature can reasonably be tested using an existing browser test, this will be undertaken before the PR is approved.
+  - Any further or additional end-to-end testing will be raised in a new issue with the 'browser test' label and referencing the original issue.
```

Comment:
Fair enough

Reply:
I thought this is what we agreed?

Reply:
Yes, we haven't added many browser tests recently, so it's been more an unspoken rule, and it's good to have it in the docs.